### ISSUE TITLE
Batch normalization on MNIST example: which is the correct parameter?/

### ISSUE THREAD

**Issue by anselmoferreira** [vlfeat/matconvnet#953]
In CNN mnist examples of matconvnet1.0-beta23, I can see the following line on cnn_mnist.m

`opts.batchNormalization = false ;`

In cnn_mnist_init.m I see the following line:

`opts.batchNormalization = true ;`

where is the update parameter? the last one, right?
